splash/network_manager.py

```diff
@@ -123,6 +123,38 @@ def createRequest(self, operation, request, outgoingData=None):
             self.log(traceback.format_exc(), min_level=1, format_msg=False)
             return super(ProxiedQNetworkAccessManager, self).createRequest(operation, request, outgoingData)
 
+    def _get_request_body(self, request, outgoing_data):
+        if outgoing_data is not None:
+            size = None
+            if outgoing_data.isSequential():
+                # In a sequential QIODevice, size() returns the value of
+                # bytesAvailable(), which is only the size of the data in the
+                # QIODevice buffer and not the total size of the output. Until
```

Comment:
Do you know if tests trigger this behavior (data with a size larger than a buffer)? It seems new tests only use very small request bodies.